content/posts/read-the-docs-2023-stats.rst

```diff
@@ -0,0 +1,109 @@
+Read the Docs 2023 Stats
+========================
+
+:date: Mar 1, 2024
+:category: Updates
+:tags: stats, year-in-review, homepage
+:authors: Eric Holscher
+:image:
+:image_credit:
+
+2023 was a steady year for Read the Docs.
+We continue to see growth in our user base,
+but our overall revenue has been relatively flat.
+Overall things are going well,
+and we're excited to continue providing a world-class documentation platform and ethical ad network for our community.
+
+.. note::
+
+	You can always see our stats for the last `30 days`_.
+
+	Our posts from 2013_, 2014_, 2015_, 2016_, 2017_, 2018_, 2019_, 2020_, 2023_ are also available.
+
+.. _Read the Docs: https://readthedocs.org/
+.. _30 days: http://www.seethestats.com/site/readthedocs.org
+.. _2013: https://blog.readthedocs.com/read-the-docs-2013-stats/
+.. _2014: https://blog.readthedocs.com/read-the-docs-2014-stats/
+.. _2015: https://blog.readthedocs.com/read-the-docs-2015-stats/
+.. _2016: https://blog.readthedocs.com/read-the-docs-2016-stats/
+.. _2017: https://blog.readthedocs.com/read-the-docs-2017-stats/
+.. _2018: https://blog.readthedocs.com/read-the-docs-2018-stats/
+.. _2019: https://blog.readthedocs.com/read-the-docs-2019-stats/
+.. _2020: https://blog.readthedocs.com/read-the-docs-2020-stats/
+.. _2023: https://blog.readthedocs.com/read-the-docs-2023-stats/
+
+
+Page views
+----------
+
+Unfortunately we don't have page view stats for 2023,
+since we decided not to migrate to Google Analytics 4.
+
+We are working to improve our stats tracking with plausible,
+but haven't fully expanded it to all our doc serving due to cost.
+
+Site stats
+----------
+
+The stats, in total numbers:
+
+* 198,645 new projects
+* 73,988 new users
+* 3,793,075 builds
```

Comment:
Previously, we were adding `(from XXXX)` on each of these numbers.

Reply:
Yea, we didn't do stats last year it seems. I can try and backport them, probably.